lib/std/fs/path.zig

```diff
@@ -468,6 +468,10 @@ fn asciiEqlIgnoreCase(s1: []const u8, s2: []const u8) bool {
 }
 
 /// On Windows, this calls `resolveWindows` and on POSIX it calls `resolvePosix`.
+/// Note: Symlinks currently dont work and should not be used,
+/// because Windows and Posix having different behavior.
+/// On Windows, symlinks are resolved after `..` is resolved.
```

Comment:
I think "Note: does not expand symlinks" is sufficient for the note here

The details on symlink resolution between platforms have to do with `realpath` and the other `std.os` calls, not this function